lib/fortnox/api/types.rb

```diff
@@ -45,9 +48,24 @@ module Types
                     .constructor(EnumConstructors.default)
 
       CountryCode = Strict::String
-                    .constrained(included_in: CountryCodes.values)
                     .optional
-                    .constructor(EnumConstructors.sized(2))
+                    .constructor do |value|
+                      next value if value.nil? || value == ''
+
+                      # Fortnox only supports Swedish translation of Sweden
+                      sweden_variants = ['se', :se, 'sweden', :sweden, 'sverige', :sverige]
+
+                      next CountryCodeString.new('SE') if sweden_variants.include?(value.downcase)
+
+                      country = ::ISO3166::Country[value] ||
+                                ::ISO3166::Country.find_country_by_name(value.to_s.downcase) ||
```

Comment:
This only works on English names and `find_country_by_translated_names` works on names given in any of the locales that you have configured, if I understand it correctly. But this data is from Fortnox that only permits names in English, unless it is Sverige which we already handle above. So the last clause here should never get called?

Also `find_country_by_name` works fine with both strings and symbols regardless of case, just like you already call `find_country_by_translated_names` 🙂

Reply:
No. Fortnox support both English and Swedish country names, except for one case: the country name of Sweden must be in Swedish.

Good catch! I'll remove the `.to_s.downcase` :)